pkg/builtinchecks/yamls/upgrade-strategy.yaml

```diff
@@ -0,0 +1,10 @@
+name: "upgrade-strategy"
+description: "Indicates when a deployment doesn't meet safe upgrade parameters"
+remediation: >-
+  Correct the update strategy to perform safe upgrades.
```

Comment:
Can we add more colour here? Safe is a very broad term. We should educate users on why the benefits of RollingUpdate, and perhaps insert a link to k8s documentation.

Reply:
I added some more detail here.  Unfortunately it doesn't appear there is a single link to the kube documentation for this because the update strategy configuration is slightly different for all objects.